create_ollama_sysext.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+export ARCH="${ARCH-x86-64}"
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 VERSION SYSEXTNAME"
+  echo "The script will download ollama release binaries (e.g., for v0.3.6) and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
+  echo "A temporary directory named SYSEXTNAME in the current folder will be created and deleted again."
+  echo "All files in the sysext image will be owned by root."
+  echo "To use arm64 pass 'ARCH=arm64' as environment variable (current value is '${ARCH}')."
+  echo "CNI version current value is 'latest'"
+  "${SCRIPTFOLDER}"/bake.sh --help
+  exit 1
+fi
+
+VERSION="$1"
+SYSEXTNAME="$2"
+
+if [ "${ARCH}" = "x86_64" ] || [ "${ARCH}" = "x86-64" ]; then
+  ARCH="amd64"
+elif [ "${ARCH}" = "aarch64" ]; then
+  ARCH="arm64"
+fi
+
+VERSION="v${VERSION#v}"
+
+BIN_URL="https://github.com/ollama/ollama/releases/download/${VERSION}/ollama-linux-${ARCH}"
+SHA_URL="https://github.com/ollama/ollama/releases/download/${VERSION}/sha256sum.txt"
+
+rm -rf "${SYSEXTNAME}"
+
+TMP_DIR="${SYSEXTNAME}/tmp"
+mkdir -p "${TMP_DIR}"
+
+curl --parallel --fail --silent --show-error --location \
```

Comment:
ollama v0.3.7 changed the artifact, are you able to update this to work with newer versions?

Reply:
Look like [`v0.3.8`](https://github.com/ollama/ollama/releases/tag/v0.3.8) just shipped as well, I'll go ahead and update the sysext to handle those.